src/tests/JIT/Regression/JitBlue/Runtime_90508/Runtime_90508.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+using Xunit;
+
+public class Runtime_90508
+{
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static Vector128<double> Div1(Vector128<double> v, double b) => 
```

Comment:
nit: why are these functions named `DivX`?

Reply:
Oops, no idea 🙂 changed to Test